packages/wdio-reporter/src/index.js

```diff
@@ -117,7 +117,6 @@ export default class WDIOReporter extends EventEmitter {
     onBeforeCommand () {}
     onAfterCommand () {}
     onScreenshot () {}
-    onRunnerStart () {}
```

Comment:
Let's keep this, it should actually be onRunnerStart instead of onStart, please review: https://github.com/webdriverio/v5/pull/35